ipm_image_node/ipm_image_node/ipm.py

```diff
@@ -23,7 +23,10 @@
 from rclpy.duration import Duration
 from rclpy.executors import MultiThreadedExecutor
 from rclpy.node import Node
-from sensor_msgs.msg import CameraInfo, Image, PointCloud2, PointField
+from sensor_msgs.msg import CameraInfo
```

Comment:
What style did you use? Because I am pretty sure it is okay to group imports in the ros2 one as long as they are in the correct order.

Reply:
This was isort with `--profile google` as you suggested.

I dont know whats right, but I like this more than the brackets () method😅

Reply:
I really hate it... it is so verbose ...

Reply:
It is at least possible in the example: https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_google.py

Reply:
https://stackoverflow.com/questions/44613886/multiple-imports-according-to-google-style-guide